packages/docs/src/components/propsTable.tsx

```diff
@@ -17,8 +17,18 @@ function dirtyMarkdown(text: string) {
         .replace(/`([^`]+)`/g, (_, code) => `<code>${code}</code>`) };
 }
 
-const renderPropRow = (prop: IPropertyEntry) => {
-    const { deprecated, documentation, inheritedFrom, internal, name, optional } = prop;
+function propsTag(intent: Intent, title: string, ...children: React.ReactNode[]) {
+    return (
+        <Tag key={title} className={Classes.MINIMAL} intent={intent}>
+            <strong>{title}</strong>
```

Comment:
Can we unbold? The color and background are already quite prominent in the table, I wouldn't say there's a need for additional cues.

With the same reasoning, can we remove the background of the `<code>` for the "Inherited" tag? Do it like `.docs-prop-type` (right now there's a vertical alignment issue but we'd be better off going with a `span.pt-monospace-text`)

<img width="332" alt="screen shot 2017-02-07 at 8 14 11 pm" src="https://cloud.githubusercontent.com/assets/199754/22723310/05a5126e-ed72-11e6-9bd8-805da4d725a2.png">

^ Notice how the prop names match 👍